### PR TITLE
fix(content): Remove reference to 'Embers' in an early Remnant mission

### PR DESCRIPTION
**Pull request by roadrunner56**
**Content (Artwork / Missions / Jobs)**

## Summary
When I played `Remnant: Logistics 1` I had not gotten very far through the Remnant storyline. My character would thus be unaware of the significance of the phrase 'the Embers' in Remnant culture, so I thought it was odd that I could sing about the Embers despite this. This PR removes that reference. 
My last PR (https://github.com/endless-sky/endless-sky/pull/10746) locked this mission behind other missions that would ensure that the player would hear the phrase 'the Embers' before they got this mission, but it was too convoluted and this seemed like the easier solution.

## Screenshots
N/A

## Testing Done
I changed a few words.

## Save File
This save file can be used to test these changes:
[warp core logistics 1 changes test~original.txt](https://github.com/user-attachments/files/17873680/warp.core.logistics.1.changes.test.original.txt)
Thanks to warp core for the save file.

### CODE REVIEW

**Review by TheGiraffe3 — APPROVED** [review]

File: data/remnant/remnant 1 introduction.txt [hunk 1]
```diff
@@ -3823,9 +3823,9 @@ mission "Remnant: Logistics 1"
 		has "Remnant: Defense 1: done"
 	on offer
 		conversation
-			`You step out of the <ship> amidst the strange sight of the <origin> spaceport. The structures that surround the landing pads look as much grown as built, and they seem to merge into the landing pads themselves. As usual, the dock entrance is busy. When your commlink vibrates, you have to find a place where the traffic eddies so you can step aside. When you do, you learn you need to check in with the information desk.`
+			`You step into the strange sight of the <origin> spaceport. The structures that surround the landing pads look as much grown as built, and they seem to merge into the landing pads themselves. As usual, the dock entrance is busy. When your commlink vibrates, you have to find a place where the traffic eddies so you can step aside. When you do, you learn you need to check in with the information desk.`
```
Comment (tibetiroka) [inline data/remnant/remnant 1 introduction.txt:3826]: What is the reason for this change? It doesn't seem to be related to the change you discussed in the description.

Comment (roadrunner56) [reply data/remnant/remnant 1 introduction.txt:3826]: Sorry, I should have edited the PR description. It's based on this suggestion from Warp: https://github.com/endless-sky/endless-sky/pull/10746#discussion_r1835809497

**Review by tibetiroka — APPROVED** [review]
